pints/_log_pdfs.py

```diff
@@ -115,6 +115,178 @@ def sample(self, n=1):
         raise NotImplementedError
 
 
+class PooledLogPDF(LogPDF):
+    r"""
+    Creates an object of :class:LogPDF, with a log-pdf which is the sum of the
```

Comment:
``` :class:`LogPdf` ```

I also wouldn't start with "Creates an object of" when describing the class itself